dask/dataframe/core.py

```diff
@@ -475,7 +478,7 @@ def map_partitions(self, func, *args, **kwargs):
         ----------
         func : function
             Function applied to each partition.
-        args, kwargs :
+        args, kwargs : Scalar, Delayed or object
```

Comment:
`args` and `kwargs` can contain these types, but aren't these types themselves. I'd remove the type note here, and move it to the description below. Something like `Arguments and keywords may contain ``Scalar``, ``Delayed``, regular python objects`.